tests/topotests/ospf6-topo1/test_ospf6_topo1.py

```diff
@@ -404,27 +411,25 @@ def test_shutdown_check_stderr():
 
 
 def test_shutdown_check_memleak():
```

Comment:
This function could be simplified to this:

```py
def test_memory_leak():
    "Run the memory leak test and report results."
    tgen = get_topogen()
    if not tgen.is_memleak_enabled():
        pytest.skip('Memory leak test/report is disabled')

    tgen.report_memory_leaks()
```

The `tgen` (topology object) has a function that calls all routers memory leak report function:
https://github.com/FRRouting/frr/blob/master/tests/topotests/lib/topogen.py#L362

Reply:
I did try to use that, but it fails if both "STDERR" and "MEMLEAK" reporting are enabled - and they are in the CI.

Reply:
That seems like a bug, I'll investigate that (nothing else to do here).